src/main/java/lotto/lotto/Lotto.java

```diff
@@ -0,0 +1,90 @@
+package lotto.lotto;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lotto implements LottoGenerator {
+
+    private static final int LOTTO_CNT = 6;
+    private static final int MAX_LOTTO_NUMBER = 45;
+
+    public final List<LottoNumber> lottoNumbers;
+
+    public Lotto() {
+        this.lottoNumbers = new ArrayList<>();
+    }
```

Comment:
이 경우 Collection타입의 자료구조들은 필드초기화를 해주는것도 좋습니다. ㅎㅎ

```suggestion
    public final List<LottoNumber> lottoNumbers = new ArrayList<>();

    public Lotto() {
    }
```

Reply:
선언할때 자료구조 필드 초기화 해주게 되면 final로 적용 시  public Lotto(List<LottoNumber> lottoNumbers) 에서 오류가 나서 이번에 final을 빼고 선언하였습니다.  이렇게 적용하는게 맞을까요!? 

public List<LottoNumber> lottoNumbers = new ArrayList<>();

    public Lotto() {
    }

    public Lotto(List<LottoNumber> lottoNumbers) {
        this.lottoNumbers = lottoNumbers;
    }